README.md

```
@@ -24,123 +24,243 @@ Running server:
 
 #### Prometheus metrics
 
-```
+Returns metrics in Prometheus format for monitoring.
+
+```http
 GET /_metrics
 ```
+
+**Example metrics:**
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)